Resources/Textures/Clothing/Shoes/Misc/clothWrap.rsi/meta.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from https://github.com/ParadiseSS13/Paradise/",
```

Comment:
Commit?

Reply:
Holy hell this is annoying, all the others PRs i made nearly has none of it, and only this one you annoy me with.

Gaaah.